Makefile

```diff
@@ -290,13 +304,18 @@ endif
 	$(N64_EMULATOR) $<
 
 
-.PHONY: all clean setup run distclean assetclean
+.PHONY: all uncompressed compressed clean setup run distclean assetclean
+.DEFAULT_GOAL := uncompressed
```

Comment:
I think `uncompressed` is already the default because it's the first goal. But why should the default be `uncompressed` instead of `all`?

Reply:
I think explicit instead of implicit is usually better.
Building a compressed rom is slow and just building an uncompressed one is usually enough to know the rom is matching or to test any changes

Reply:
I mean, the idea of `all` is to build `all`, and most of the time you won't want to build a compressed rom, so that's why it isn't the default.
At some point in MM we built the compressed rom by default, and it was painfully slow for getting work done, so we changed it to build the uncompressed rom by default

Reply:
This is fine by me now that the "default" output rom is uncompressed